sinks/cortex/cortex.go

```diff
@@ -73,11 +85,13 @@ func ParseConfig(config interface{}) (veneur.MetricSinkConfig, error) {
 }
 
 // NewCortexMetricSink creates and returns a new instance of the sink
-func NewCortexMetricSink(URL string, timeout time.Duration, proxyURL string, name string) (*CortexMetricSink, error) {
+func NewCortexMetricSink(URL string, timeout time.Duration, proxyURL string, logger *logrus.Entry, name string, tags map[string]string) (*CortexMetricSink, error) {
 	return &CortexMetricSink{
 		URL:           URL,
 		RemoteTimeout: timeout,
 		ProxyURL:      proxyURL,
+		tags:          tags,
+		logger:        logger.WithFields(logrus.Fields{"sink": name, "sink_type": "cortex"}),
```

Comment:
Veneur already adds a `metric_sink` field with the sink name to the logger here:
https://github.com/stripe/veneur/blob/master/server.go#L362
Does this get overwritten?